runtime-react/pom.xml

```diff
@@ -0,0 +1,197 @@
+                        </goals>
+                        <phase>generate-resources</phase>
+                        <configuration>
+                            <escapeString>\</escapeString>
+                            <resources>
+                                <resource>
+                                    <directory>${project.basedir}/src/main/codestarts</directory>
+                                    <includes>
+                                        <include>**/codestart.yml</include>
+                                    </includes>
+                                    <filtering>true</filtering>
+                                </resource>
+                                <resource>
+                                    <directory>${project.basedir}/src/main/codestarts</directory>
+                                    <excludes>
+                                        <exclude>**/codestart.yml</exclude>
+                                    </excludes>
+                                    <filtering>false</filtering>
+                                </resource>
+                            </resources>
+                            <outputDirectory>${project.build.directory}/filtered-codestarts/codestarts</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>generate-codestart-jar</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <configuration>
+                            <classesDirectory>${project.build.directory}/filtered-codestarts</classesDirectory>
+                            <includes>
+                                <include>codestarts/**</include>
+                            </includes>
+                            <excludes>
+                                <exclude>codestarts/**/hilla-scaffold-package-name-holder.txt</exclude>
+                            </excludes>
+                            <classifier>codestarts</classifier>
+                            <skipIfEmpty>true</skipIfEmpty>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+        </plugins>
+    </build>
+    <profiles>
+        <profile>
+            <id>update-hilla-codestart</id>
+            <build>
+                <defaultGoal>org.codehaus.mojo:build-helper-maven-plugin:bsh-property@hilla-scaffold</defaultGoal>
+                <plugins>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>3.4.0</version>
```

Comment:
move version to property. Should also be done for other versions if some are there?

Reply:
Also pluginManagement could be clever

Reply:
The plugin was already in pluginManagement. Removed the version